lib/views/invoice_slack_view.ex

```diff
@@ -4,10 +4,15 @@ defmodule Aprb.Views.InvoiceSlackView do
   def render(event) do
     partner_data = fetch_partner_data(event["properties"]["partner_id"])
     %{
-      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      text: ":money_with_wings: Invoice #{event["object"]["display"]}",
       attachments: "[{
                       \"fields\": [
                         {
+                          \"title\": \"Artworks\",
+                          \"value\": \"#{artworks_display_from_artworkgroups(event["properties"]["artwork_groups"])}\",
+                          \"short\": false
+                        },
+                        {
                           \"title\": \"Total\",
```

Comment:
Yes! @gnilekaw also brought the same point up in https://github.com/artsy/aprb/pull/84, we ended up doing this during hackthon and never had a chance to change it but it makes total sense, doing it in another PR 👍